### PR TITLE
TW-1644: allow access to the gallery is not center

### PR DESCRIPTION
**Pull request by sherlockvn**
### MERGE THIS FIRST:
https://github.com/linagora/linagora-design-flutter/pull/44

### Related ticket: 
https://github.com/linagora/twake-on-matrix/issues/1644

### Android:

https://github.com/linagora/twake-on-matrix/assets/43041967/e4eb9390-234a-40d4-bd0e-81d4eb29fcac

### IOS:

https://github.com/linagora/twake-on-matrix/assets/43041967/c6113142-1b8e-4fe9-857b-bc2ab2ea5c95

### CODE REVIEW

**Review by nqhhdev — APPROVED** [review]

**Comment by Te-Z** [comment]
Quick win to improve UX: IMO the bottom sheet should close when clicking on the "go to settings" button. This way we sugget to the user that he must retry to send something from his gallery

**Comment by sherlockvn** [comment]
> Quick win to improve UX: IMO the bottom sheet should close when clicking on the "go to settings" button. This way we suggest to the user that he must retry to send something from his gallery

if user change any permission, the app will be restart and the state of app will lost. so IMO, you don't have to worry about this case

**Review by hieutbui — APPROVED** [review]

**Comment by Te-Z** [comment]
> > Quick win to improve UX: IMO the bottom sheet should close when clicking on the "go to settings" button. This way we suggest to the user that he must retry to send something from his gallery
> 
> if user change any permission, the app will be restart and the state of app will lost. so IMO, you don't have to worry about this case

Not every time. Look at your demo video

**Comment by sherlockvn** [comment]
> > > Quick win to improve UX: IMO the bottom sheet should close when clicking on the "go to settings" button. This way we suggest to the user that he must retry to send something from his gallery
> > 
> > 
> > if user change any permission, the app will be restart and the state of app will lost. so IMO, you don't have to worry about this case
> 
> Not every time. Look at your demo video

Okay i fixed it

https://github.com/linagora/twake-on-matrix/assets/43041967/22001a6b-62c0-4ba6-b87e-99f595ac7567

**Comment by Te-Z** [comment]
Hi @sherlockvn did you push your changes ?

**Comment by sherlockvn** [comment]
> Hi @sherlockvn did you push your changes ?

sr i pushed again now

File: lib/presentation/mixins/single_image_picker_mixin.dart [hunk 1]
```diff
@@ -37,18 +38,20 @@ mixin SingleImagePickerMixin on CommonMediaPickerMixin {
         expandedWidget: const SizedBox(height: 50),
```
Comment (Te-Z) [inline lib/presentation/mixins/single_image_picker_mixin.dart:38]: can you put this in the style class please ?

**Review by Te-Z — APPROVED** [review]

File: lib/presentation/mixins/single_image_picker_mixin.dart (hunk 1 above)
Comment (Te-Z) [inline lib/presentation/mixins/single_image_picker_mixin.dart:38]: LGTM overrall. Just add this to style class.
Linagora-design-flutter PR merged

Comment (sherlockvn) [reply lib/presentation/mixins/single_image_picker_mixin.dart]: done

**Review by nqhhdev — APPROVED** [review]
LGTM

**Review by Te-Z — APPROVED** [review]

**Pull request merged**